.github/workflow_scripts/env_setup.sh

```diff
@@ -46,6 +46,10 @@ function install_features {
     python3 -m pip install --upgrade -e features/
 }
 
+#function install_eda {
```

Comment:
General comments about CI script. Remember to reactivate them :)

Reply:
Done